Resources/public/style/httplug.css

```diff
@@ -80,6 +80,11 @@
     font-size: 12px;
 }
 
+.httplug-duration {
+    min-width: 6ch;
```

Comment:
1 ch is the width of a "0" in the current font. It looks easier to me as I want the box at least large as if it was filled with 6 characters.